check-with-vale.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${GITHUB_BASE_REF}" ]
+    then 
+        MAINBRANCH="origin/main"
+    else
+        MAINBRANCH="origin/$GITHUB_BASE_REF"
+fi
+
+FILES=$(git diff --name-only --diff-filter=AM "$MAINBRANCH")
+
+if [ -n "${FILES}" ]
+    then
+        echo "Validating languages on file added or modified in comparison to $MAINBRANCH with $(vale -v)"
+        echo ""
+        echo "==============================================================================================================================="
+        echo "Read about the error terms that cause the build to fail at https://redhat-documentation.github.io/vale-at-red-hat/docs/reference-guide/termserrors/"
+        echo "==============================================================================================================================="
+        echo ""
+        set -x
+        vale ${FILES} --minAlertLevel=error --glob='*.adoc'
```

Comment:
```suggestion
        vale ${FILES} --minAlertLevel=error --glob='*.adoc'
        vale ${FILES} --minAlertLevel=error --glob='*.adoc' --output=json > vale_errors.json
```

Reply:
Or if we don't want to display error in readable format we could skip the other command and only do it in json,

Reply:
We want the build error to be nice and readable in the travis log though don't we?